src/expandableCalendar/WeekCalendar/index.tsx

```diff
@@ -70,14 +70,15 @@ const WeekCalendar = (props: WeekCalendarProps) => {
           }) :
           sameWeek(item, date, firstDay));
       if (pageIndex !== currentIndex.current) {
+        const adjustedIndexFrScroll = APPLY_ANDROID_FIX ? NUM_OF_ITEMS - 1 - pageIndex : pageIndex;
```

Comment:
'shouldUseAndroidRTLFix' should be the name, and we need to take it from the constants